watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/biomembrane_filtration/tests/test_multi_sweep.py

```diff
@@ -0,0 +1,34 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+
+import os
+import pytest
+from watertap.examples.flowsheets.case_studies.wastewater_resource_recovery.biomembrane_filtration import (
+    multi_sweep,
+)
+
+sweep_list = []
+for case_num in [1, 2, 3, 4]:
+    sweep_list.append(case_num)
+
+
+@pytest.mark.parametrize("case_num", sweep_list)
+@pytest.mark.integration
+def test_multi_sweep(case_num, tmp_path):
+    cwd = os.getcwd()
+    os.chdir(tmp_path)
+    nx = 11
```

Comment:
I suggest changing `nx=1` in the test to make it run more quickly.

Reply:
I thik `nx` represents the number of points we take for each variable, if `nx = 1`, then we only take 3 points. Since I don't test combination of multiple variables, it is pretty fast for running this test